prog/mutation.go

```diff
@@ -426,18 +502,41 @@ func (ma *mutationArgs) collectArg(arg Arg, ctx *ArgCtx) {
 		if typ.Kind == BufferString && len(typ.Values) == 1 {
 			return // string const
 		}
+		if typ.Kind == BufferFilename || typ.Kind == BufferText {
+			priority = 60
+			break
+		}
+		priority = 85
 	case *PtrType:
 		if arg.(*PointerArg).IsSpecial() {
 			// TODO: we ought to mutate this, but we don't have code for this yet.
 			return
 		}
+		priority = 40
+	case *ProcType, *VmaType, *ResourceType:
+		priority = 50
+	case *IntType:
+		if typ.RangeBegin == 0 && typ.RangeBegin == typ.RangeEnd { // no range assigned
+			priority = maxPrio
+			break
+		}
+
+		priority = float64((typ.RangeEnd - typ.RangeBegin)) / float64(^uint64(0)) * maxPrio
```

Comment:
This seems to give extremely low probability to bool (effectively "never"). I don't think the relation between range and priority should be linear because we will never enumerate all values for even a single uint64 (if always mutate just that 1 value).